intellij-plugin-structure/structure-youtrack/src/main/kotlin/com/jetbrains/plugin/structure/youtrack/YouTrackPluginManager.kt

```diff
@@ -0,0 +1,114 @@
+package com.jetbrains.plugin.structure.youtrack
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.jetbrains.plugin.structure.base.decompress.DecompressorSizeLimitExceededException
+import com.jetbrains.plugin.structure.base.plugin.*
+import com.jetbrains.plugin.structure.base.problems.*
+import com.jetbrains.plugin.structure.base.utils.*
+import com.jetbrains.plugin.structure.youtrack.bean.YouTrackAppManifest
+import com.jetbrains.plugin.structure.youtrack.problems.createIncorrectYouTrackPluginFileError
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+
+class YouTrackPluginManager private constructor(private val extractDirectory: Path) : PluginManager<YouTrackPlugin> {
+
+  companion object {
+    const val DESCRIPTOR_NAME = "manifest.json"
+
+    private val LOG: Logger = LoggerFactory.getLogger(YouTrackPluginManager::class.java)
```

Comment:
By project convention, logs are top-level variables.

Reply:
Thanks. Fixed.
Please address the same project convention violation in your other plugin managers:
- `com.jetbrains.plugin.structure.edu.EduPluginManager`
- `com.jetbrains.plugin.structure.fleet.FleetPluginManager`
- `com.jetbrains.plugin.structure.hub.HubPluginManager`
- `com.jetbrains.plugin.structure.teamcity.TeamcityPluginManager`
- `com.jetbrains.plugin.structure.toolbox.ToolboxPluginManager`